go.mod

```diff
@@ -464,3 +579,5 @@ replace (
 	// See https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/12d41f40b0d408b0167633d8095160d3343d46ac/go.mod#L38
 	github.com/openshift/api v3.9.0+incompatible => github.com/openshift/api v0.0.0-20180801171038-322a19404e37
 )
+
+replace github.com/dop251/goja => github.com/andrewkroh/goja v0.0.0-20190128172624-dd2ac4456e20
```

Comment:
I don't think relying on a fork from an employee's personal GitHub account is the right approach. I'd prefer to have these under Elastic's organization, as it would be easier to manage and maintain in the long run. According to https://github.com/elastic/beats/issues/41084, we don't want to keep temporary replacements around for too long. We should make our best effort to upstream any fixes or switch to/maintain another up-to-date fork

Reply:
100% true. But blocking this PR for something that is already true for beats, doesn't provide forward progress. Lets push for progress and make adjustments a long the way.

Reply:
The sarama one was alreay there.  Added fsnotify, we won't need it until we bring in Auditbeat, but be might as well start the pain now so we have more motivation to get those changes pushed upstream.